src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Telemetry/TelemetryThresholds.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Razor.Workspaces.Telemetry;
+
+/// <summary>
+/// A set of constants used to reduce the telemetry emitted to the set that help us understand
+/// which LSP is taking the most time in the case that the overall call is lengthy.
+/// </summary>
+internal class TelemetryThresholds
+{
+    internal const int CodeActionRazorTelemetryThresholdMS = 2000;
```

Comment:
Consider making these static readonly fields of type `TimeSpan` rather than creating a `TimeSpan` at every usage. For example:

```suggestion
    public static readonly CodeActionRazorTelemetryThreshold = TimeSpan.FromMilliseconds(2000);
```

If it's necessary to get at the real millisecond value, that's easy enough to do from the `TimeSpan`.

Reply:
Done